_inc/client/components/jetpack-notices/index.jsx

```diff
@@ -160,11 +160,7 @@ export const UserUnlinked = React.createClass( {
 			this.props.connectUrl &&
 			this.props.siteConnected
 		) {
-			const text = __( 'You, %(userName)s, are not connected to WordPress.com.', {
-				args: {
-					userName: this.props.username
```

Comment:
Since this is no longer used, all its references either direct or indirect should be removed, for example
`username: getUsername( state ),`